en/micro-integrator/docs/overview/about-this-release-7.1.0.md

```diff
@@ -305,7 +305,29 @@ The following features, which are available in ESB runtimes, are removed from th
 			Rule Mediator
 		</td>
 		<td>
-			The rule mediator has been removed from Ei 7.1.0 as it is not widely used.
+			The rule mediator has been removed from EI 7.1.0 as it is not widely used.
+		</td>
+		<td>
+			-
+		</td>
+	</tr>
+	<tr>
+		<td>
+			HL7 Console
+		</td>
+		<td>
+			The HL7 Console, which was available in the management console of previous ESBs, is removed in EI 7.x as it is not widely used.
+		</td>
```

Comment:
I'm wondering whether we should explicitly mention this, as HL7 Message Store removal will imply this

Reply:
I thought the HL7 console might have more use cases than the message store.

Reply:
not really, its coupled with store only afaik

Reply:
updated as per suggestion